apps/fishing-map/features/workspace/vessels/VesselTranslations.ts

```diff
@@ -43,4 +43,9 @@ const translations = 'used_to_get_dynamic_translations_autogenerated'
 // t('vessel.gearTypes.trap net', 'Trap Net')
 // t('vessel.gearTypes.purse seine', 'Purse Seine')
 
+// Belize gear types
```

Comment:
we'll need to run the i18n-parser and merge the translations.json source file before the release. what about doing it separately of this PR?

Reply:
sure, makes sense